File: app/cpp/console/code/GPerson.cpp
//===============================================
#include "GPerson.h"
//===============================================
GPerson::GPerson(const std::string& name, const int& code, const double& coef) {
    m_name = name;
    m_code = code;
    m_coef = coef;
}
//===============================================
GPerson::~GPerson() {
    
}
//===============================================
void GPerson::print() {
    printf("Personne[name] : %s\n", m_name.c_str());
    printf("Personne[code] : %d\n", m_code);
    printf("Personne[coef] : %.2f\n", m_coef);
}
//===============================================

File: app/php/code/GPerson.php
<?php
class GPerson {
    //===============================================
    private $name;
    private $code;
    private $coef;
    //===============================================
    public function __construct($name, $code, $coef) {
        $this->name = $name;
        $this->code = $code;
        $this->coef = $coef;
    }
    //===============================================
    public function print() {
        echo sprintf("Person[Nom] : %s<br/>\n", $this->name);
        echo sprintf("Person[Code] : %d<br/>\n", $this->code);
        echo sprintf("Person[Coef] : %.2f<br/>\n", $this->coef);
    }
    //===============================================
}
?>
